frontend/src/metabase/home/components/EmbedMinimalHomepage/EmbedMinimalHomepage.tsx

```diff
@@ -0,0 +1,73 @@
+import { jt, t } from "ttag";
+import { Button, Card, Flex, Icon, Stack, Text, Title } from "metabase/ui";
+import Link from "metabase/core/components/Link";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { getDocsUrl } from "metabase/selectors/settings";
+import { useSelector } from "metabase/lib/redux";
+import { removeShowEmbedHomepageFlag } from "./util";
+import { NumberedList } from "./EmbedMinimalHomepage.styled";
+
+type EmbedMinimalHomepageProps = {
+  onDismiss: () => void;
+};
+
+export const EmbedMinimalHomepage = ({
+  onDismiss,
+}: EmbedMinimalHomepageProps) => {
+  const learnMoreUrl = useSelector(s =>
+    // eslint-disable-next-line no-unconditional-metabase-links-render -- this is only visible to admins
+    getDocsUrl(s, { page: "embedding/start" }),
+  );
+
+  const dismiss = () => {
+    removeShowEmbedHomepageFlag();
+    onDismiss();
+  };
+
+  return (
+    <Stack>
+      <Text
+        fw="bold"
+        color="text-dark"
+      >{t`Get started with Embedding Metabase in your app`}</Text>
+      <Card px={40} py={32} maw={570}>
+        <Stack spacing="lg">
+          <Flex justify="space-between">
+            <Title size="h4">{t`As you expressed interest in Embedding, follow these steps to start`}</Title>
+
+            <Button
+              variant="white"
+              size="small"
+              color="text-dark"
+              p={0}
+              onClick={dismiss}
+            >
+              <Icon name="close" />
+            </Button>
+          </Flex>
+
+          <NumberedList>
+            <li>{t`Enable and configure embedding in settings`}</li>
+            <li>
+              {t`Select or create a dashboard or question to do a static embed`}
+            </li>
+            <li>{t`Follow the quickstart to do an interactive embed`}</li>
+          </NumberedList>
+
+          <Stack spacing="sm">
+            <Link to="/admin/settings/embedding-in-other-applications">
+              <Button
+                variant="filled"
+                leftIcon={<Icon name="gear" />}
+              >{t`Get started`}</Button>
+            </Link>
+
+            <Text>{jt`${(
+              <ExternalLink href={learnMoreUrl}>{t`Learn more`}</ExternalLink>
```

Comment:
Components inside `jt` needs to have `key` because `jt` will convert these to array as far as I understand.

This is the error from this component.
![image](https://github.com/metabase/metabase/assets/1937582/78a66fd8-2895-4ccd-be3a-8fa218313892)

Reply:
I keep forgetting about it 🤦  bdb803dfc3776a07662b0a6f98db53d16701d51e